charts/cinder-csi-plugin/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: latest
 description: Cinder CSI Chart for OpenStack
 name: openstack-cinder-csi
-version: 1.4.8
+version: 1.5.0
```

Comment:
As this change is not backward compatible, I think we should bump the major version, i.e. 2.0.0. If the users have already installed the previous cinder-csi-plugin chart, they should first `helm uninstall` before installing 2.0.0, right?

Reply:
@lingxiankong done